apps/backend/src/database/helpers.ts

```diff
@@ -0,0 +1,8 @@
+import { LoggerOptions } from 'typeorm';
+import { DatabaseLogger } from './database-logger';
+
+export const loggerOptions: LoggerOptions = ['query', 'error', 'schema'];
+export const logLevels =
+  process.env.DB_LOGS_ENABLED === 'true' ? loggerOptions : undefined;
```

Comment:
why undefined?

Reply:
The Base Datasource logger options does not include false as a type, but since the logger itself is optional we can intentionally set this to undefined in the datasource config.